releasenotes/notes/2.1/deprecate-library-circuits-4f8b234500f90d50.yaml

```diff
@@ -2,7 +2,7 @@ deprecations_circuits:
   - |
     The circuit library underwent a refactoring in the Qiskit 1.3 release,
     in which alternatives for objects of type :class:`.QuantumCircuit` were provided
-    that are either a :class:`.Instruction` or a Python function for construction. This refactoring
+    that are either an :class:`.Instruction` or a Python function for construction. This refactoring
```

Comment:
"and the migration guide for more details" -- can that link to the migration guide?

Reply:
Apparently, there is no migration guide per se so I refined the note accordingly.